packages/AutoTDD-Tests.package/ATDDNullDataSourceTests.class/instance/testSingleton.st

```diff
@@ -0,0 +1,7 @@
+tests
+testSingleton
+	
+	| dataSource2 |
```

Comment:
Maybe there is a better name than `dataSource2`?

Reply:
Ahhhhh dataSource3 :D